src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    static final String BALL = "Ball1";
+    static final String BALL2 = "Ball2";
+    static final String BALL3 = "Ball3";
+    static final String[] FINAL_NUMBER = {BALL, BALL2, BALL3};
```

Comment:
Why do you need this?

Reply:
Because I thought I need to make 3 constants , to test the method.